pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -77,6 +79,39 @@ def _gen_distributor():
     }
 
 
+def _sync_and_publish_repo(server_config, importer=None, distributors=None):
```

Comment:
Is there any reason to make this function as generic as it is? This function is generic, so it's not going to get widespread use. In addition, it is used only once:

```python
repo, tasks = _sync_and_publish_repo(
    cls.cfg,
    importer=importer_config
)
```

I ask because, in my opinion, it is a liability to add in capabilities that are not needed or used. How about changing this function signature to the following?

```python
def _sync_and_publish_repo(server_config, importer):
```